### PR TITLE
Issue #444: Use inspect.stack()[0].function in debug() in order to retrieve function name.

### LINKED ISSUE #444: Use `inspect.stack()[0].function` in debug() in order to retrieve function name.

### PR DESCRIPTION
**Pull request by amalota**
closes #444

### CODE REVIEW

File: git_machete/client.py [hunk 1]
```diff
@@ -414,7 +415,7 @@ def get_root_of(branch: LocalBranchShortName) -> LocalBranchShortName:
                     "cycle in the resulting graph or the candidate is a stale branch"))
             if upstream:
                 debug(
-                    "discover_tree()",
+                    f"{inspect.stack()[0].function}()",
```
Comment (PawelLipski) [inline git_machete/client.py]: Hmmm that's pretty ugly... would it be possible to move that logic to `debug` itself? 🤔

File: git_machete/client.py [hunk 2]
```diff
@@ -2096,8 +2097,8 @@ def create_github_pr(
                 opt_yes=False)
 
         current_user: Optional[str] = git_machete.github.derive_current_user_login()
-        debug(f'create_github_pr({head})', f'organization is {org}, repository is {repo}')
-        debug(f'create_github_pr({head})', 'current GitHub user is ' + (current_user or '<none>'))
+        debug(f"{inspect.stack()[0].function}({head})", f'organization is {org}, repository is {repo}')
+        debug(f"{inspect.stack()[0].function}({head})", 'current GitHub user is ' + (current_user or '<none>'))
```
Comment (PawelLipski) [inline git_machete/client.py]: (at least move method name extraction to `debug`... and if possible, also the params!)

**Comment by amalota** [comment]
e.g. run
image

**Review by PawelLipski — APPROVED** [review]
:tada: Good change aka _dobra zmiana_!

**Pull request merged**